src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketsId;
+    private Long userId;
```

Comment:
Checklist: When choosing the fields to be returned in your ShoppingCartResponseDto, keep in mind that cartId and userId hold the same value.

Reply:
You can just remove  userId because it is same as ShoppingCart id